src/App.module.scss

```diff
@@ -1,38 +1,25 @@
-.App {
-  text-align: center;
-}
-
-.appLogo {
-  height: 40vmin;
-  pointer-events: none;
+.navbar ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .appLogo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.navbar li {
+  float: left;
```

Comment:
floats are typically a bit of a pain - there are normally easier ways.
What were you trying to achieve?